full/frontend/components/Header.js

```diff
@@ -0,0 +1,78 @@
+import styled from "styled-components";
+
+let HeaderRoot = styled.div`
+  position: absolute;
+  width: 100%;
+  color: #586e75;
+  box-sizing: border-box;
+`;
+
+let HeaderTop = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 0 72px;
+  min-height: 80px;
+  background-color: #002b36;
+  box-sizing: border-box;
+`;
+
+let HeaderBottom = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  min-height: 40px;
+  background-color: #c4c4c4;
+  box-sizing: border-box;
+`;
+
+let HeaderMiddleCell = styled.div``;
+
+const Header = props => {
+  const filters = props.filters;
+  const reset = props.reset;
+  const apply = props.apply;
+  const setCheckedItems = props.setCheckedItems;
+  const checkedItems = props.checkedItems;
+  const statusMessage = props.statusMessage;
+
+  const handleChangeFilter = event => {
+    setCheckedItems({
+      ...checkedItems,
+      [event.target.name]: event.target.checked
+    });
+  };
+
+  let filterEls = filters.map(item => {
+    return (
+      <label key={item.label}>
+        {item.label}
+        <input
+          type="checkbox"
+          id={item.label}
+          name={"filter_" + item.label.toLowerCase()}
+          onChange={handleChangeFilter}
+          checked={checkedItems["filter_" + item.label.toLowerCase()]}
+        />
+      </label>
+    );
+  });
+
+  return (
+    <HeaderRoot>
+      <HeaderTop>
```

Comment:
Instead of HeaderTop / HeaderMiddleCell / HeaderBottom, what do you think about more semantic names like HeaderLogo / HeaderControls / HeaderStatus? 

The design of this little app probably won't change much, so this doesn't really matter that much. But still gotta speak from my long-ingrained aversion to class names based on layout…

Reply:
I tweaked the names a bit; this is going to get revamped in a bit anyway.